.env

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
It's not advisable to commit the .env file to Github. Instead, create a `.sample.env` file that has dummy data that resemble the .env file.

Please remove this file from your git history. Add it to a `.gitignore` file.

Reply:
To do this successfully:
- Add the name of the file to `.gitignore` file
- On your terminal use the commands:
 > git rm -r --cached .
 > git add
 > git commit -m "remove .env file from git"